CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased] - TBD
 
+## [4.3.4] - 2022-01-10
+**Note that this release bumps the WordPress minimum version from 4.0 to 5.7 and the PHP minimum version from 5.6 to 7.4.**
+
+### Added
+- Add automated PHPCS scanning (props [@csloisel](https://github.com/csloisel), [@Sidsector9](https://github.com/Sidsector9) via [#169](https://github.com/10up/windows-azure-storage/pull/169)).
```

Comment:
This doesn't seem to be what #169 is and I can't seem to find a merged PR that adds PHPCS scanning, but perhaps I'm missing something here if you can help clarify?

Reply:
This should have been #170, fixed that